sig-and-wg/sig-core/proposals/authorization-proxy.md

```diff
@@ -0,0 +1,40 @@
+# Installation on managed cluster
+
+> Created on 2018-08-02 by Piotr Bochyński (@pbochynski).
+
+## Status
+
+Proposed
+
+## Abstract
+
+It is not possible to install kyma project on managed clusters like Google Kubernetes Engine (GKE) or Azure Kubernetes Service (AKS). To enable kyma-project adoption it has to run on any infrastructure supporting Kubernetes. Kyma requires access to kube-apiserver configuration and uses some kubernetes alpha features, that is **not allowed** in many managed clusters. Even if some providers allow for such configuration, it has to happen during cluster provisioning, what makes kyma installation process more complex.
```

Comment:
Please apply those changes for this section, there are some minor changes within the text:

`It is not possible to install the Kyma project on the managed clusters such as Google Kubernetes Engine (GKE) or Azure Kubernetes Service (AKS). To enable the Kyma project adoption, it must run on any infrastructure supporting Kubernetes. Kyma requires access to the kube-apiserver configuration and uses some Kubernetes alpha features, which is **not allowed** in many managed clusters. Even if some providers allow such configuration, it must happen during the cluster provisioning, which makes Kyma installation process more complex.`

Reply:
done